app/services/short_url_service.rb

```diff
@@ -0,0 +1,28 @@
+require "json"
+
+class ShortUrlService
+  include HTTParty
+  base_uri "https://api.short.io/"
```

Comment:
nit: Should we place this in some sort of constants module? Perhaps something like [this](https://stackoverflow.com/questions/47153612/best-way-to-define-constants-in-ruby-ror-at-one-place)?

Reply:
I like that idea. I'll look into it

Reply:
resolved in 96eb0b0
let me know if this is what you were thinking of.

Reply:
LGTM!